src/it/banduplicate-classes-ignore-when-identical/pom.xml

```diff
@@ -0,0 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
This integration test uses the new mode (ignoreWhenIdentical=true) to verify the new code doesn't fail the build when the same exact class appears on the classpath multiple times. Meaning, duplicates classes are ignored when they're indentical.